content/spaces/cursors.textile

```diff
@@ -0,0 +1,164 @@
+
+The following is an example of a member setting their cursor position by adding an event listener to obtain their cursor coordinates and then publishing their position using the @set()@ method:
+
+```[javascript]
+window.addEventListener('mousemove', ({ clientX, clientY }) => {
+  space.cursors.set({ position: { x: clientX, y: clientY }, data: { color: 'red' } });
+});
+```
+
+The following is an example payload of a cursor event. Cursor events are uniquely identifiable by the @connectionId@ of a cursor.
+
+```[json]
+{
+  "hd9743gjDc": {
+    "connectionId": "hd9743gjDc",
+    "clientId": "clemons#142",
+    "position": {
+      "x": 864,
+      "y": 32
+    },
+    "data": {
+      "color": "red"
+    }
+  }
+}
+```
+
+The following are the properties of a cursor event payload:
+
+|_. Property |_. Description |_. Type |
+| connectionId | The unique identifier of the member's "connection":/connect. | String |
+| clientId | The "client identifier":/auth/identified-clients for the member. | String |
+| position | An object containing the position of a member's cursor. | Object |
+| position.x | The position of the member's cursor on the X-axis. | Number |
+| position.y | The position of the member's cursor on the Y-axis. | Number |
+| data | An optional arbitrary JSON-serializable object containing additional information about the cursor, such as a color. | Object |
+
+h2(#subscribe). Subscribe to cursor events
+
+Subscribe to cursor events by registering a listener. Cursor events are emitted whenever a member moves their cursor by calling @set()@. Use the @subscribe()@ method on the @cursors@ object of a space to receive updates.
+
+<aside data-type='note'>
+<p>The rate at which cursor events are published is controlled by the @outboundBatchInterval@ property set in the "cursor options":#options of a space.</p>
+</aside>
+
+The following is an example of subscribing to cursor events:
+
+```[javascript]
+space.cursors.subscribe('update', (cursorUpdate) => {
+  console.log(cursorUpdate);
+});
+```
+
+h3(#unsubscribe). Unsubscribe from cursor events
+
+Unsubscribe from cursor events to remove previously registered listeners.
+
+The following is an example of removing a single listener for cursor events:
+
+```[javascript]
+space.cursors.unsubscribe(`update`, listener);
+```
+
+Or remove all listeners:
+
+```[javascript]
+space.cursors.unsubscribe();
+```
+
+h2(#options). Cursor options
+
+Cursor options are set at a space level when creating or retrieving a space. They are used to control the behavior of live cursors.
+
+The following cursor options can be set:
+
+- outboundBatchInterval := The interval, in milliseconds, at which a batch of cursor positions are published. This is multiplied by the number of members in a space, minus 1. The default value is 100ms.
+- paginationLimit := The number of pages searched from history for the last published cursor position. The default is 5.
+
+h2(#retrieve). Retrieve cursor events
```

Comment:
For all the retrieve sections, it would be useful to share an example payload (in other pages too). The structure is slightly different as it's a full set rather than a single diff update.